app/components/primer/underline_panel.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= wrapper(with_panel: true, **@wrapper_arguments) do %>
+  <%= render Primer::BaseComponent.new(**@system_arguments) do %>
+    <% if @align == :right %>
+      <%= actions %>
+    <% end %>
+
+    <%= render body do %>
+      <% tabs.each do |tab| %>
+        <%= tab %>
+      <% end %>
+    <% end %>
+
+    <% if @align == :left %>
+      <%= actions %>
+    <% end %>
+  <% end %>
```

Comment:
This is exactly the same as `UnderlineNav`, so I wonder if we should call it here instead of duplicating the logic

Reply:
Hmmm, this view template is the same but the logic determining how these should be rendered lives within each component.

We **could** pull this view into a partial and render it from each component view file.

Reply:
Wouldn't it be possible for `UnderlinePanel` to call `UnderlineNav` here with the correct parameters?

Reply:
Ah, I get what you're saying! How would we initialize the slots for `UnderlineNav` through `UnderlinePanel`?

Reply:
for example, how would we initialize the tab slot for `UnderlineNav` through `UnderlinePanel` with the correct arguments?

Reply:
kinda hacky but this works https://github.com/primer/view_components/commit/a81ba64dc6fe27978941ad1d138e0885c4e5e25d

I'm kinda "linking" the slots from both components, so calling `.tab` on Panel, will actually call it in Nav

Reply:
cc @BlakeWilliams not the first time I want a parent's slot to be redirected to a child's slot